improver/cli/combine.py

```diff
@@ -60,17 +61,19 @@ def process(
             +, -, \*, add, subtract, multiply, min, max, mean
         new_name (str):
             New name for the resulting dataset.
-        broadcast_to_threshold (bool):
-            If True, broadcast input cubes to the threshold coord prior to combining -
-            a threshold coord must already exist on the first input cube.
+        broadcast (str):
+            If specified, the input cubes will be broadcast over the coordinate name provided. If
+            "threshold" is provided the plugin will try to find a threshold coordinate on the
+            probability cube.
         minimum_realizations (int):
             If specified, the input cubes will be filtered to ensure that only realizations that
             include all available lead times are combined. If the number of realizations that
             meet this criteria are fewer than this integer, an error will be raised.
         cell_method_coordinate (str):
             If specified, a cell method is added to the output with the coordinate
             provided. This is only available for max, min and mean operations.
-
+        expand_bound (bool):
```

Comment:
Should this be `expand_bounds` (plural) throughout?

Also, I'm not sure how this works. Default is `True`, but I think a CLI argument `--expand-bound` will also set it to `True`. Please can you add an example to the acceptance tests to show how this can be set to `False`?

Reply:
I've added in the acceptance test to test the expand_bound option as you suggest as well as the appropriate acceptance test data.

In regards to whether the argument should be plural, ideally it would be but there is already a function in Improver called expand_bounds and I was trying to avoid renaming that function and changing a few different plugins and tests. That being said if you felt it would be beneficial it is doable?